supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -37,7 +37,7 @@ This renders as:
 Create an Ansible inventory file that is named `/_<path>_/inventory/hosts`.
 ====
 
-To italicize a user-replaced value in a code block, you must add an attribute to apply text formatting, such as `subs="+quotes"` or `subs="normal"`, to the attribute list of the code block.
+To italicize a user-replaced value in a code block, you must add a substitution attribute to apply text formatting to the attribute list of the code block. You can use `subs="+quotes"`,  `subs="normal"`, or `subs="attributes+"`.
```

Comment:
@adellape I thought that `subs="attributes+"` was for if you're replacing an attribute in a code block. Like {product-title} and wanting it to resolve to "OpenShift Container Platform". Unless I'm mistaken, I don't think that you can use `subs="attributes+"` to italicize in a code block.

Reply:
Sorry you're right, I started conflating too many things when looking up examples using multiple styles per block. I'll rewind the `attributes+`-related changes.

Though I do notice now that neither the Supplementary Guide or the [openshift-docs guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc) mention `attributes+` for the attribute-replacement use case. I imagine other CCS teams might be using attributes in a similar way, but I'm not sure... If they do, maybe worth a follow-up PR for the SG.